File: Backend/SharedLib/ResponseModel.cs
namespace SharedLib
{
    public class ResponseModel
    {
        public bool Success { get; set; }
        public string Message { get; set; } = "";
        public object? Data { get; set; } 
        public ResponseModel(bool success,string message,object? data =null) {
            this.Success = success;
            this.Message = message;
            this.Data = data;
        }
    }
}

File: Frontend/lms-public/src/util/types/responseModel.ts
export class ResponseModel {
  constructor(
    success: boolean = false,
    message: string = "",
    data: any = null
  ) {
    this.success = false;
    this.message = "";
    this.data = null;
  }

  success: boolean;
  message: string;
  data: any;
}
